src/page-overview/assets/init.js

```diff
@@ -4,6 +4,7 @@ import {
   setCurrentState,
 } from "./data-forms";
 import { Slider } from "./sliders";
+import initTooltips from "./tooltip";
```

Comment:
nit: Let's try to be consistent about import/exporting names!

- Can you convert the existing files to using default exports if you prefer it?
- The default export should have the same name as the file. (Note to self: I should add `eslint-plugin-filenames`'s `match-exported` rule.)

Reply:
Created #10 to add the `match-exported` rule!